docs/data/pulsar.yml

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 ################################################################################
 
-version: 4.1.0
+version: 4.2-SNAPSHOT
```

Comment:
I think we don't change this until we do a release. `-SNAPSHOT` is meaningless before it's release. This field will be used to generate docs which is only valid for formal releases.

Reply:
```suggestion
version: 4.1.10
```